neps/nep-0463.md

```diff
@@ -0,0 +1,174 @@
+```
+  NEP: 463
+  Title: Decentralized Autonomous Organization Standard
+  Author: DAO Builders Group
+  Status: Draft
+  DiscussionsTo: nearbuilders.com/tg-dao
+  Type: Standards
+  Created: 21-Feb-2023
+```
+
+## Summary
+
+A standard interface to support decentralized autonomous organizations (DAOs) that enable proposals with governance.
+
+## Motivation
+
+There is a need for **interoperable DAO systems** on NEAR and other blockchains.
+
+Essentially, this NEP requests feedback from the Contract Standards Working Group.
+
+### Benefits:
+
+* greater accessibility
+* better coordination
+* easier development
+
+#### *What use cases does it support?*
+
+* *complete display of “My DAOs” in wallets and on-chain profiles*
+* *composability across platforms with a common ABI for any `DAO`*
+* *cohesive, yet decentralized governance by NEAR communities*
+
+DAO contract standards help builders create new DAOs that integrate with existing DAOs.
+
+## Rationale
+
+[Sputnik DAOs](https://github.com/near-daos/sputnik-dao-contract) represent a widely used contract system on NEAR, and the code has been thoroughly tested and reviewed by the community. Our hope is to improve accessibility of that robust open-source product by formally introducing a compatible standard.
+
+Considering the principles of [EIP-4824](https://daostar.one/EIP), let's assume DAOs have at least two primitives:
+
+1. **MEMBERSHIP** is defined by a set of addresses.
+2. **BEHAVIOR** is defined by a set of possible contract actions, including calls to external contracts and calls to internal functions.
+
+**PROPOSALS** relate membership and behavior; they are *objects* with which members can interact.
+
+* If and when executed, they become actions of the DAO (via governance).
```

Comment:
I could not see it as a rationale for a DAO standard. I encourage you to read other NEPs (e.g. [here is a small one](https://github.com/near/NEPs/blob/master/neps/nep-0330.md)).